packages/DOS/README

```diff
@@ -1,7 +1,10 @@
 Gisle Vanem made curl build fine on DOS (and MingW) with djgpp, OpenSSL and his
 Watt-32 stack.
 
-'make djgpp' in the root curl dir should build it fine.
+'make -f Makefile.dist djgpp-clean djgpp' in the root curl dir should build it fine.
```

Comment:
I think this is your personal preference I don't see why we should put this upstream. In other words `make djgpp` is sufficient. You could add that `make djgpp clean` or whatever works to clean incomplete builds.